.circleci/config.yml

```diff
@@ -48,3 +57,7 @@ workflows:
           filters:
             tags:
               only: /(helm-)?[0-9]+(\.[0-9]+)*(-[a-z]+)?/
+      - e2e-testing:
+          filters:
+            tags:
+              only: /(helm-)?[0-9]+(\.[0-9]+)*(-[a-z]+)?/
```

Comment:
newline please! unless it upsets CircleCI?

Reply:
Is the filter there to be conservative about when the end-to-end testing is run, for now?

Reply:
Actually this doesn't seem to filter what CircleCi does -- the e2e-testing jobs is run for this PR. So I have misunderstood what this means. What does it mean?

Reply:
Yeah that filter does nothing, CircleCI config so fun 😞 

Looks like a branch filter with exclude all is required:

```yaml
  filters:
    tags:
      only: /(helm-)?[0-9]+(\.[0-9]+)*(-[a-z]+)?/
    branches:
      ignore: /.*/
```

Reply:
Removing this will disable builds for tags. https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag

Reply:
Ahhh ok I'll put it back